plugin/sdk_v2/extension/decorators.py

```
@@ -0,0 +1,71 @@
+extension = _ExtensionDecorators()
+
+__all__ = [
+    "EXTENSION_ENTRY_META",
+    "EXTENSION_HOOK_META",
+    "ExtensionEntryMeta",
+    "ExtensionHookMeta",
+    "_not_impl",
+    "extension_entry",
+    "extension_hook",
+    "extension",
+]
```

Ruff (0.15.5)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)